main.dart

```diff
@@ -1,3 +1,58 @@
 void main() {
-  print('hello');
+  int firstDetail = findDetail(inputData, '*', '.');
+  int secondDetail = findDetail(inputData, ':', null);
+  print(firstDetail);
+  print(secondDetail);
+
+  int phoneCount = createPhone(firstDetail, secondDetail);
+  print('Phone created is $phoneCount');
+}
+
+const inputData = [
+  '*.**::::**..*::.*…**.*::',
+  '*.**::::**..*::.*…**.*::',
+  '*.**::::**..*::.*…**.*::',
+];
+
+int findDetail(List<String> inputData, String firstDetailPart, String? secondDetailPart) {
```

Comment:
rename firstDetailPart and secondDetailPart 
they called the same as in main function, but have constantly other logic